mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -267,7 +267,27 @@ class LinearSVC @Since("2.2.0") (
       if (featuresStd(i) != 0.0) rawCoefficients(i) / featuresStd(i) else 0.0
     }
     val intercept = if ($(fitIntercept)) rawCoefficients.last else 0.0
-    copyValues(new LinearSVCModel(uid, Vectors.dense(coefficientArray), intercept))
+    val vec = Vectors.dense(coefficientArray)
+    return createModel(dataset, vec, intercept, objectiveHistory)
```

Comment:
I'd like: `createModel(dataset, Vectors.dense(coefficientArray), intercept, objectiveHistory)`
var `vec` and `return` are unnecessary